### PR TITLE
feat: world manifest

### PR DESCRIPTION
**Pull request by aleortega**

### CODE REVIEW

File: src/entities/CheckScenes/task/taskRunnerSqs.ts [hunk 1]
```diff
@@ -43,6 +48,50 @@ const placesAttributes: Array<keyof PlaceAttributes> = [
   "textsearch",
 ]
 
+const ACCESS_KEY = env("AWS_ACCESS_KEY")
+const ACCESS_SECRET = env("AWS_ACCESS_SECRET")
+const BUCKET_HOSTNAME = env("BUCKET_HOSTNAME")
+const BUCKET_NAME = env("AWS_BUCKET_NAME", "")
+
+async function updateWorldManifest() {
```
Comment (braianj) [inline src/entities/CheckScenes/task/taskRunnerSqs.ts:56]: the name `updateWorldManifest ` is a bit tricky, it's about world, or a genesis city or both?

File: src/entities/CheckScenes/RoadCoordinates.json
Comment (braianj) [inline src/entities/CheckScenes/RoadCoordinates.json:1]: WDYT about putting this in `src/__data__`

File: src/entities/CheckScenes/task/taskRunnerSqs.ts [hunk 2]
```diff
@@ -43,6 +48,50 @@ const placesAttributes: Array<keyof PlaceAttributes> = [
...
+    if (BUCKET_HOSTNAME) {
+      url.hostname = BUCKET_HOSTNAME
+    }
+
+    return url.toString()
+  })
+
+  const worldManifestPositions = await calculateWorldManifestPositions()
+
+  return await fetch(signedUrl, {
```
Comment (braianj) [inline src/entities/CheckScenes/task/taskRunnerSqs.ts:86]: should we validate this `signedUrl` ?

File: src/entities/CheckScenes/utils.ts [hunk 3]
```diff
@@ -88,3 +90,26 @@ export async function getWorldAbout(
   const worldContentServer = await ContentServer.getInstanceFrom(url)
   return worldContentServer.fetch(`/world/${worldName}/about`)
 }
+
+export async function calculateWorldManifestPositions() {
+  const occupiedPositions = await PlacePositionModel.find({})
+  const emptyPositions = []
+
+  for (let x = -150; x <= 150; x++) {
+    for (let y = -150; y <= 150; y++) {
```
Comment (braianj) [inline src/entities/CheckScenes/utils.ts:99]: We need to review this, since we can go to coordinates `162.153`

**Review by braianj — CHANGES_REQUESTED** [review]

File: src/entities/CheckScenes/task/taskRunnerSqs.ts (hunk 1 above)
Comment (aleortega) [reply src/entities/CheckScenes/task/taskRunnerSqs.ts]: There is no ubiquitous language yet for this but some teammates from E@ are referring to this manifest JSON as `World Manifest`.  Anyways, I've renamed it to `Genesis City Manifest`.

File: src/entities/CheckScenes/task/taskRunnerSqs.ts (hunk 2 above)
Comment (aleortega) [reply src/entities/CheckScenes/task/taskRunnerSqs.ts:87]: What should we validate? 🤔

File: src/entities/CheckScenes/task/taskRunnerSqs.ts [hunk 4]
```diff
@@ -43,6 +49,53 @@ const placesAttributes: Array<keyof PlaceAttributes> = [
   "textsearch",
 ]
 
+const ACCESS_KEY = env("AWS_ACCESS_KEY")
+const ACCESS_SECRET = env("AWS_ACCESS_SECRET")
+const BUCKET_HOSTNAME = env("BUCKET_HOSTNAME")
+const BUCKET_NAME = env("AWS_BUCKET_NAME", "")
+
+async function updateGenesisCityManifest() {
```
Comment (braianj) [inline src/entities/CheckScenes/task/taskRunnerSqs.ts:57]: Can we move this function to a different file?

**Review by braianj — APPROVED** [review]
Beautiful!

File: src/entities/CheckScenes/task/taskRunnerSqs.ts (hunk 2 above)
Comment (braianj) [reply src/entities/CheckScenes/task/taskRunnerSqs.ts]: if there is a signedUrl

File: src/entities/CheckScenes/task/taskRunnerSqs.ts (hunk 4 above)
Comment (aleortega) [reply src/entities/CheckScenes/task/taskRunnerSqs.ts:57]: Sure thing!

**Pull request merged**